exporter/googlecloudpubsubexporter/exporter.go

```diff
@@ -0,0 +1,70 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package googlecloudpubsubexporter
+
+import (
+	"context"
+
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/consumer"
+	"go.opentelemetry.io/collector/consumer/pdata"
+	"go.uber.org/zap"
+)
+
+const name = "googlecloudpubsub"
+
+// pubsubExporter is a wrapper struct of OT cloud trace exporter
```

Comment:
Is this comment accurate? I don't see any dependencies on the OT cloud trace exporter, will they be introduced in a future PR?

Reply:
Oeps, removed comment. Old comment, from the start of the implementation